### ISSUE TITLE
2.5 shows new Dashboard UI page

### ISSUE THREAD

**Issue by sowmyav27** [rancher/dashboard#6650]
**on 2.5.16-rc2**

- Navigating to Continuous delivery shows the new Dashboard UI.
<img width="1063" alt="Screen Shot 2022-08-12 at 7 17 14 AM" src="https://user-images.githubusercontent.com/26032343/184396458-91d1951f-75f6-48d7-af2b-9db079f1a85d.png">

**Comment by catherineluse** [comment]
I asked Nancy about this - Basically, this is expected behavior because the Ember UI never had a Fleet page. It's not a regression because it always linked to the dashboard. Nancy linked this commit https://github.com/rancher/ui/commit/d7436d4059fc438ce94838b79601b180cd38af17#diff-25f95243c321a3888504101fe93345c3780c6319602aed49d44fc239a4e41706 which shows dashboard links being added to Ember in v2.5.0.

**Comment by catherineluse** [comment]
For reference, this screenshot shows the expected result:

![Uploading Screen Shot 2022-08-12 at 1.53.35 PM.png…]()

This is the Continuous Delivery link that is leading to v2.6 code even though it's a v2.5 RC:
<img width="729" alt="Screen Shot 2022-08-12 at 12 13 46 PM" src="https://user-images.githubusercontent.com/20599230/184451870-c7e00563-161f-49b1-b044-17def2707b26.png">

**Comment by catherineluse** [comment]
Update: It turned out that the bug was caused by `v2.5.16-rc2` being cut from the wrong UI branch. We expect the bug to be fixed in RC3 with no code changes, just with the RC being cut from the right branch.

**Comment by gaktive** [comment]
For QA, once RC3 for 2.5.16 is in place, this should be fixed. Let us know if you see anything out of place.

**Comment by brudnak** [comment]
# :white_check_mark: PASSED

## Reproduction Environment

| Component                 | Version / Type                           |
| ------------------------- | ---------------------------------------- |
| Rancher version           | 2.5.16-rc2                               |
| Installation option       | Helm                                     |
| RKE binary version used   | v1.2.20                                  |
| If Helm Chart k8s cluster | v1.20.15                                 |
| Cert Details              | Let's Encrypt                            |
| Helm version              | v2.16.8-rancher1                         |
| Downstream cluster type   | Not Applicable                           |
| Downstream K8s version    | Not Applicable                           |
| Logged in user role       | Administrator                            |
| Browser type              | Google Chrome                            |
| Browser version           | 103.0.5060.134 (Official Build) (x86_64) |

**ADDITIONAL SETUP**

- Helm commands for Rancher installation:

```shell
helm install rancher rancher-latest/rancher \
  --namespace cattle-system \
  --set hostname=$URL_VAR \
  --set ingress.tls.source=letsEncrypt \
  --set letsEncrypt.email=$EMAIL_VAR \
  --set letsEncrypt.ingress.class=nginx \
  --version 2.5.16-rc2
```

## Reproduction steps

1. Starting from the **default** homepage `/g/clusters`
2. Hover over `Tools` ==> click `Continuous Delivery`
3. Observe the page you are routed to
  
## Additional Info

## RESULTS

#### :white_check_mark: Expected
Expect to be routed to the "old" dashboard

#### :x: Actual
Routed to the new Dashboard UI

<hr>
  
## Validation Environment

| Component                 | Version / Type                           |
| ------------------------- | ---------------------------------------- |
| Rancher version           | 2.5.16-rc3                               |
| Installation option       | Helm                                     |
| RKE binary version used   | v1.2.20                                  |
| If Helm Chart k8s cluster | v1.20.15                                 |
| Cert Details              | Let's Encrypt                            |
| Helm version              | v2.16.8-rancher1                         |
| Downstream cluster type   | Not Applicable                           |
| Downstream K8s version    | Not Applicable                           |
| Logged in user role       | Administrator                            |
| Browser type              | Google Chrome                            |
| Browser version           | 103.0.5060.134 (Official Build) (x86_64) |

**ADDITIONAL SETUP**

- Helm commands for Rancher installation:

```shell
helm install rancher rancher-latest/rancher \
  --namespace cattle-system \
  --set hostname=$URL_VAR \
  --set ingress.tls.source=letsEncrypt \
  --set letsEncrypt.email=$EMAIL_VAR \
  --set letsEncrypt.ingress.class=nginx \
  --version 2.5.16-rc3
```

## Validation steps

1. Starting from the **default** homepage `/g/clusters`
2. Hover over `Tools` ==> click `Continuous Delivery`
3. Observe the page you are routed to
  
## Additional Info

## RESULTS

#### :white_check_mark: Expected
Expect to be routed to the "old" dashboard

#### :white_check_mark:Actual
Routed to the "old" dashboard for CD
